internal/gatewayapi/route.go

```diff
@@ -1009,14 +1010,44 @@ func (t *Translator) processDestination(backendRef v1beta1.BackendRef,
 
 	var endpoints []*ir.DestinationEndpoint
 	switch KindDerefOr(backendRef.Kind, KindService) {
-	// TODO: Use EndpointSlice for ServiceImport
 	case KindServiceImport:
-		backendIps := resources.GetServiceImport(backendNamespace, string(backendRef.Name)).Spec.IPs
-		for _, ip := range backendIps {
-			ep := ir.NewDestEndpoint(
-				ip,
-				uint32(*backendRef.Port))
-			endpoints = append(endpoints, ep)
+		serviceImports := resources.GetServiceImport(backendNamespace, string(backendRef.Name))
```

Comment:
shouldnt this be a single `serviceImport` ?

Reply:
yeah it is a tpyo :/